src/app/visual/responsive/responsive-container.component.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'sky-responsive-container',
+  templateUrl: './responsive-container.component.html',
+  styleUrls: ['./responsive-container.component.scss']
+})
+export class SkyResponsiveContainerComponent {}
```

Comment:
This demo works great, but I was a little confused at first - why did you call this a "container" component? Is it really a container? It seems more like a component that demonstrates responsive properties. Is there a better name we could give this that would let a new dev know what its intent is?

Reply:
Done